front-end/src/app/player/player.component.css

```diff
@@ -0,0 +1,21 @@
+.player-card {
+    background-color: #e4f7e9;
+    padding: 25px;
+    margin: 15px;
+    margin-left: 0px;
+    width: fit-content;
+}
+
+.contents {
```

Comment:
`contents` and `image` are quite generic names, and as these CSS classes apply globally, can we namespace these classes or give them more unique names?